compiler/qsc/src/interpret/tests.rs

```diff
@@ -2089,5 +2089,30 @@ mod given_interpreter {
                 entry: None,
             }
         }
+
+        #[test]
+        fn name_resolution_from_source_named_main_should_succeed() {
+            let sources = SourceMap::new(
+                [(
+                    "Main".into(),
```

Comment:
Shouldn't this be `Main.qs` for the implicit namespace to be correctly parsed? (I guess I'm wrong because clearly this code compiles?)

Reply:
The source map is expected to have already dropped the ".qs" extension, so that's why it's missing here.